server/src/main/java/io/druid/server/http/HistoricalResource.java

```diff
@@ -52,4 +52,16 @@ public Response getLoadStatus()
   {
     return Response.ok(ImmutableMap.of("cacheInitialized", coordinator.isStarted())).build();
   }
+
+  @GET
+  @Path("/loadStatusCode")
```

Comment:
(Optional) How about using the endpoint `/readiness`. It's short, all small-caps, and relevant to its usage.

Reply:
Changed